int8 flag                       #标志位
int32 roadid                     #界面重启
int32 stationid 		 #紧急停车按钮
